SparkPost/2web2ui
# SA-1680: Parse query param for query_filters

### What Changed
 - Parses for `query_filters` and returns with helper function
 - Updates `query_filters` in URL query params

### How To Test
 - Add console.log to parseSearchNew to check `ret.queryFilters`
 - Use param `?query_filters=%7B%22groupings%22%3A%5B%7B%22AND%22%3A%7B%22domains%22%3A%7B%22eq%22%3A%5B%22gmail.com%22%2C%22yahoo.com%22%2C%22hotmail.com%22%5D%2C%22like%22%3A%5B%22mail%22%5D%7D%2C%22sending_domains%22%3A%7B%22not%22%3A%7B%22eq%22%3A%5B%22sparkpost.com%22%5D%7D%7D%7D%7D%2C%7B%22OR%22%3A%7B%22templates%22%3A%7B%22eq%22%3A%5B%22default%22%5D%7D%2C%22campaigns%22%3A%7B%22like%22%3A%5B%22333%22%2C%22344%22%2C%22355%22%5D%7D%7D%7D%2C%7B%22OR%22%3A%7B%22campaigns%22%3A%7B%22not%22%3A%7B%22like%22%3A%5B%22Friday%22%5D%2C%22eq%22%3A%5B%22stuff%22%2C%22things%22%5D%7D%7D%7D%7D%5D%7D`
 - Verify that object is parsed correctly

### To Do
- [ ] Swap out parseSearch for parseSearchNew where used in report builder

### Notes
- A bit dependent on SA-1664 to have new storage available.
